src/blkdrv/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+if(WITH_BLUESTORE)
+set(libblkdrv_srcs
+    BlockDevice.cc)
+endif(WITH_BLUESTORE)
+
+if(HAVE_LIBAIO OR HAVE_POSIXAIO)
+  list(APPEND libblkdrv_srcs
+    kernel_drv/KernelDevice.cc
+    kernel_drv/io_uring.cc
+    ceph_aio/aio.cc)
```

Comment:
Perhaps take this opportunity  to rename aio.cc to ceph_aio.cc to match the include files that was already renamed to avoid a nameing conflict.
That way things are consistent.

Reply:
@wjwithagen 
If it needs to rename aio.cc, then the io_uring.cc also needs to be renamed since there's header file ceph_io_uring.h
@tchaikov 
Do you think it's necessary to  rename aio.cc to be ceph_aio.cc?

Reply:
@wjwithagen 
I've followd your suggestion to change 1)aio.cc -> ceph_aio.cc  2) io_uring.cc -> ceph_io_uring.cc